it/test/controllers/NextUpdatesControllerISpec.scala

```diff
@@ -301,9 +301,7 @@ class NextUpdatesControllerISpec extends ControllerISpecHelper {
 
         Then("the quarterly updates info sections")
         res should have(
-          elementTextBySelector("#one-year-opt-out-message")(expectedValue = "You are currently reporting quarterly on a " +
-            "voluntary basis for the 2021 to 2022 tax year. You can choose to opt out of quarterly updates and " +
-            "report annually instead.")
+          elementTextBySelector("#one-year-opt-out-message")(expectedValue = "You are currently reporting quarterly on a voluntary basis for the 2021 to 2022 tax year. You can choose to opt out of quarterly updates and report annually instead.")
```

Comment:
I know this was pre-existing but think we should be testing content in the views instead of the IT tests tbh. 

We can move these checks to the views instead and for more scenarios, easier to test there I think